src/content/pages/programme/sessions.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Sessions Preview EuroPython 2025
+subtitle: This is a preliminary list of the confirmed sessions for EuroPython 2025. The list will get modified until the final schedule is published.
+---
+
+# Sessions preview
+
+The following sessions are a preliminary list of the confirmed proposals that
+will be part of the conference. The list will keep growing in the following
+days and some sessions might be replaced in case of them being withdrawn by the
+authors.
+
+Go to session type:
+* [Keynotes](#keynote)
```

Comment:
Just to confirm that this is intended: "Keynotes" (plural) <> "Keynote" (singular)

Reply:
Yeah, it's just an anchor to the section that was called Keynote. I used Singular in all the types :|